msr-flows/flow-tests/input.py

```diff
@@ -0,0 +1,135 @@
+# Imports
+from time import time, sleep
+import serpent_builder as sb
+import complex_serpent_builder as csb
+import restart_serpent_builder as rsb
+import os
+import res_dep_analysis as rda
+import subprocess
+import os.path
+from os import path
+
+# User Definitions
+INPUT_NAME = 'cycle_test'
+DIR_NAME = 'msr_cycle_test'
+NUM_CYCLES = 3
+CYCLE_TIME_SECONDS = 2
+CYCLE_STEP_SIZE_SECONDS = 1
+OUTPUT_NAME = 'output'
+PLOTTING = True
+NON_CYCLE = False
+MULTI_CORE = False
+RESTART_CYCLE = True
+
+# Functions
+
+
+def run_script(INPUT_NAME, OUTPUT_NAME, input_script):
+    '''
+    Writes the input_script string to a file with the INPUT_NAME
+    which is run and outputs to the OUTPUT_NAME.
+    '''
+    with open(INPUT_NAME, 'w+') as input_file:
+        input_file.write(input_script)
+    os.system('./sss2 -omp 32 ' + str(INPUT_NAME) + ' > ' + str(OUTPUT_NAME))
+    return
+
+
+def check_wrk_file(INP_NAME):
+    '''
+    Allows the script to continue once the .wrk file is generated
+    '''
+    wrk_name = str(INP_NAME) + '.wrk'
+    while not path.exists(wrk_name):
+        sleep(20)
+    return
+
+
```

Comment:
Avoid doing this.
Define a main:
``` if __name__ == "__main__": ```

And move the code in there

Reply:
I am not sure exactly what you mean by this. Do you mean the code that isn't in functions should be in `if __name__ == "__main__"`? I just want to check before I make the change.

Reply:
Yes!